src/direct-auth-idx/direct-auth-idx/Models/EnrollPhoneViewModel.cs

```diff
@@ -12,7 +12,6 @@ public class EnrollPhoneViewModel
         [Required]
         [Display(Name = "Phone number")]
         [DataType(DataType.PhoneNumber)]
```

Comment:
Why the regex validation was removed?

Reply:
The expected error message comes from the server.  The regex prevented the phone number from being submitted.  May be worth talking about more; I did it primarily to record the json response for a bad phone number.